includes/pages/admin_import.php

```diff
@@ -382,7 +382,7 @@ function shift_sort($a, $b) {
 }
 
 function export_xls() {
-	$filename = $fn = tempnam ('/tmp', '.csv'); //  Temporary File Name
+	$filename = tempnam('/tmp', '.csv');//  Temporary File Name
```

Comment:
@DishantK1807 Don't delete the space between the code and the comment. That space is to separate code and comment and make it easier to read the code. In some coding convention, people even require 2 spaces here. But Drupal doesn't mention, I will skip it now. But next time, please keep space between code and comment (if those are on the same line).